Makefile

```diff
@@ -159,106 +92,51 @@ rm -rf $$TMP_DIR ;\
-
-# .PHONY: bundle-push
-# bundle-push: ## Push the bundle image.
-# 	$(MAKE) docker-push IMG=$(BUNDLE_IMG)
-
-# .PHONY: opm
-# OPM = ./bin/opm
-# opm: ## Download opm locally if necessary.
-# ifeq (,$(wildcard $(OPM)))
-# ifeq (,$(shell which opm 2>/dev/null))
-# 	@{ \
-# 	set -e ;\
-# 	mkdir -p $(dir $(OPM)) ;\
-# 	OS=$(shell go env GOOS) && ARCH=$(shell go env GOARCH) && \
-# 	curl -sSLo $(OPM) https://github.com/operator-framework/operator-registry/releases/download/v1.15.1/$${OS}-$${ARCH}-opm ;\
-# 	chmod +x $(OPM) ;\
-# 	}
-# else
-# OPM = $(shell which opm)
-# endif
-# endif
-
-# # A comma-separated list of bundle images (e.g. make catalog-build BUNDLE_IMGS=example.com/operator-bundle:v0.1.0,example.com/operator-bundle:v0.2.0).
-# # These images MUST exist in a registry and be pull-able.
-# BUNDLE_IMGS ?= $(BUNDLE_IMG)
-
-# # The image tag given to the resulting catalog image (e.g. make catalog-build CATALOG_IMG=example.com/operator-catalog:v0.2.0).
-# CATALOG_IMG ?= $(IMAGE_TAG_BASE)-catalog:v$(VERSION)
-
-# # Set CATALOG_BASE_IMG to an existing catalog image tag to add $BUNDLE_IMGS to that image.
-# ifneq ($(origin CATALOG_BASE_IMG), undefined)
-# FROM_INDEX_OPT := --from-index $(CATALOG_BASE_IMG)
-# endif
-
-# # Build a catalog image by adding bundle images to an empty catalog using the operator package manager tool, 'opm'.
-# # This recipe invokes 'opm' in 'semver' bundle add mode. For more information on add modes, see:
-# # https://github.com/operator-framework/community-operators/blob/7f1438c/docs/packaging-operator.md#updating-your-existing-operator
-# .PHONY: catalog-build
-# catalog-build: opm ## Build a catalog image.
-# 	$(OPM) index add --container-tool docker --mode semver --tag $(CATALOG_IMG) --bundles $(BUNDLE_IMGS) $(FROM_INDEX_OPT)
-
-# # Push the catalog image.
-# .PHONY: catalog-push
-# catalog-push: ## Push a catalog image.
-# 	$(MAKE) docker-push IMG=$(CATALOG_IMG)
-
-
-
 all build: build-diskmaker build-operator
 .PHONY: all build
 
 build-diskmaker:
-	env GOOS=$(TARGET_GOOS) GOARCH=$(TARGET_GOARCH) go build -mod=vendor -ldflags '-X main.version=$(REV)' -o $(TARGET_DIR)/diskmaker $(CURPATH)/diskmaker_manager
+	env GOOS=$(TARGET_GOOS) GOARCH=$(TARGET_GOARCH) go build -mod=vendor -ldflags '-X main.version=$(REV)' -o $(TARGET_DIR)/diskmaker $(CURPATH)/cmd/diskmaker-manager
+.PHONY: build-diskmaker
 
 build-operator:
-	env GOOS=$(TARGET_GOOS) GOARCH=$(TARGET_GOARCH) go build -mod=vendor -ldflags '-X main.version=$(REV)' -o $(TARGET_DIR)/local-storage-operator $(CURPATH)
+	env GOOS=$(TARGET_GOOS) GOARCH=$(TARGET_GOARCH) go build -mod=vendor -ldflags '-X main.version=$(REV)' -o $(TARGET_DIR)/local-storage-operator $(CURPATH)/cmd/local-storage-operator
+.PHONY: build-operator
 
 images: diskmaker-container operator-container must-gather
+.PHONY: images
 
-push: images push-images
-
-push-images:
-	docker push ${DISKMAKER_IMAGE}
-	docker push ${OPERATOR_IMAGE}
-	docker push ${MUST_GATHER_IMAGE}
+push: images
```

Comment:
This will trigger must-gather image build from `Dockerfile.mustgather` which refers to an image not available on arm:

```
docker pull registry.ci.openshift.org/ocp/4.15:cli
cli: Pulling from ocp/4.15
no matching manifest for linux/arm64/v8 in the manifest list entries
```

The image (`registry.ci.openshift.org/ocp/4.15:must-gather`) that was referenced in removed `Dockerfile.mustgather.rhel7` works fine but I don't know if they're same or not. Maybe the one tagged `must-gather` is a replacement for the `cli` one?

Reply:
Since this is the only image having issues cross compiling, and we don't normally need to rebuild it, I dropped it as a requirement for `make bundle`. So the bundle will just use `quay.io/openshift/origin-local-storage-mustgather:latest` for the mustgather image (same as `hack/sync_bundle` did).

Reply:
As far as `4.15:cli` vs `4.15:must-gather` goes, it should be using `4.15:cli` as the base image. The ART bot changed the base image to `cli` back in https://github.com/openshift/local-storage-operator/commit/03f0cf29c4d19ff01cda8ca5350dc137872ed3b8 -- Dockerfile.mustgather.rhel7 just never got updated because it's not used or maintained by ART.